contracts/apps/group/GroupApp.sol

```diff
@@ -0,0 +1,74 @@
+pragma solidity 0.4.15;
+
+import "../App.sol";
+
+import "../../common/Initializable.sol";
+import "../../common/IForwarder.sol";
+import "../../common/EVMCallScript.sol";
+
+contract GroupApp is App, Initializable, IForwarder, EVMCallScriptRunner {
+    mapping (address => bool) members;
+    string context;
+
+    event AddMember(address indexed entity);
+    event RemoveMember(address indexed entity);
+
+    bytes32 constant public ADD_MEMBER_ROLE = bytes32(1);
+    bytes32 constant public REMOVE_MEMBER_ROLE = bytes32(2);
+
+    modifier onlyMember {
```

Comment:
I see this modifier is only used once on L58, is there a reason it's a modifier?

Reply:
Not anymore. There used to be more functions that used it.